File: app/src/main/java/com/yuaihen/wcdxg/net/exception/MyException.java
package com.yuaihen.wcdxg.net.exception;

/**
 * Created by Yuaihen.
 * on 2019/7/8
 * 处理异常
 * getMessage() 获取异常信息
 */
public class MyException extends RuntimeException {

//    public MyException() {
//    }

    public MyException(String message) {
        super(message);
    }


}

File: app/src/main/java/com/yuaihen/wcdxg/net/exception/MyException.kt
package com.yuaihen.wcdxg.net.exception

import java.lang.RuntimeException

/**
 * Created by Yuaihen.
 * on 2019/7/8
 * 使用统一的异常处理类
 */
class MyException(val errorCode: Int, val errorMsg: String) : RuntimeException(errorMsg) {


}